lib/controllers/post_controller.dart

```diff
@@ -23,22 +23,22 @@ class PostController with ChangeNotifier {
   }
 
   void switchOrg() {
-    print(posts);
     posts.clear();
     getPosts();
   }
 
   // void : function to get all Posts
   Future<void> getPosts() async {
-    final DateTime d1 = DateTime.now();
+    //final DateTime d1 = DateTime.now();
     final String currentOrgID = await preferences.getCurrentOrgId();
     final String currentUserID = await preferences.getUserId();
     this.currentUserID = currentUserID;
     final String query = Queries().getPostsById(currentOrgID);
     final Map result = await apiFunctions.gqlquery(query);
-    print(DateTime.now().difference(d1));
+
     if (result != null) {
-      print(posts.isEmpty);
+      debugPrint(posts.isEmpty.toString());
```

Comment:
This is a much needed change. It is not advisable to have print statement in production stage of the app.

Reply:
I assume there is an issue regarding this. I hope the changes I have made solves them.